airflow/providers/amazon/aws/executors/ecs/utils.py

```diff
@@ -46,6 +46,7 @@
     "assign_public_ip": "False",
     "platform_version": "LATEST",
     "check_health_on_startup": "True",
+    "adopt_task_instances": "False",
 }
 
 
```

Comment:
The Changes below are just sorting those dict keys to make it easier to read; no logic changes, just a little housekeeping while I was in there.